pnpm-lock.yaml

```diff
@@ -1,7 +1,7 @@
-lockfileVersion: '6.1'
+lockfileVersion: '6.0'
 
 settings:
-  autoInstallPeers: false
+  autoInstallPeers: true
```

Comment:
Sneaky. Gotta check out why these have been made and what they mean

Reply:
I guess that's because depdendabot runs with pnpm v8